src/store/reducers/hackathonReducer.js

```diff
@@ -26,46 +26,47 @@ const reducer = (state = initialState, action) => {
         hackDetails: {
           ...state.hackDetails,
           hasUserLiked: true,
-          numberLikes: action.payload
-        }
+          numberLikes: action.payload,
+        },
       };
     case ActionType.FETCH_ASSIGNED_HACKS:
       return {
         ...state,
-        assignedHacks: [...action.payload]
+        assignedHacks: [...action.payload],
       };
     case ActionType.FETCH_HACK_A_THON:
       // console.log("in reducer");
       // console.log(action.payload[0].fields);
       return {
         ...state,
-        items: { ...action.payload[0].fields }
+        items: { ...action.payload[0].fields },
       };
     case ActionType.FETCH_HACK_DETAILS:
       return {
         ...state,
-        hackDetails: { ...action.payload }
+        hackDetails: { ...action.payload },
       };
     case ActionType.JOIN_HACK:
       return {
         ...state,
-        hackDetails: { ...action.payload }
+        hackDetails: { ...action.payload },
       };
     case ActionType.UPDATE_HACK:
       return {
         ...state,
-        hackDetails: { ...action.payload }
+        hackDetails: { ...action.payload },
       };
     case ActionType.CLEAR_HACK:
       return {
         ...state,
-        hackDetails: null
+        hackDetails: null,
       };
     case ActionType.SET_SUBMISSION_DATA:
-      return{
+      //console.log(action.payload);
+      return {
         ...state,
-        submission: { ...action.payload }
-      }
+        submission: action.payload,
```

Comment:
you've collapsed this down to just assigning `submission` rather than spreading into a new object (good), but haven done that on a lot of the others for `assignedHacks`, `hackDetails` and `items`

Reply:
Yeah that was just how I was taught, but I can see and appreciate your point, if it's just one field then just give it an object instead of spreading it.

Reply:
it wasnt so much that i mind which way you go, it was that its inconsistent in this file.